poetry/console/commands/shell.py

```diff
@@ -0,0 +1,24 @@
+from .venv_command import VenvCommand
+
+
+class ShellCommand(VenvCommand):
+    """
+    Spawns a shell within the virtual environment.
+
+    shell [options]
+    """
+
+    help = """The <info>shell</info> command spawns a shell, according to the
+<comment>$SHELL</> environment variable, within the virtual environment.
```

Comment:
Should this be `</comment>` not `</>`?

Reply:
I initially thought so, too. But this is used extensively throughout the project and seems to work fine :)